client.go

```diff
@@ -414,6 +427,7 @@ func (c *Client) Pull(ctx context.Context, ref string, opts ...RemoteOpts) (Imag
 	return img, nil
 }
 
+// Push uplaods the provided content to a remote
```

Comment:
I don't think we refer to remotes as registries in containerd, i could be wrong on that.

@dmcgowan ?